### PR TITLE
SUS-5128 | use GoogleMaps API key in Places extension

### PR DESCRIPTION
**Pull request by macbre**
https://wikia-inc.atlassian.net/browse/SUS-5128

Avoid the following:

```
Google Maps JavaScript API warning: NoApiKeys https://developers.google.com/maps/documentation/javascript/error-messages#no-api-keys
```

Additionally, track when Google Maps API is loaded via `$.loadGoogleMaps` (uses by Places extension):

```
Wikia.Tracker: trackingevent googlemaps/open/apiloaded/ [analytics track]
```

### CODE REVIEW

File: extensions/wikia/Places/PlacesController.class.php [hunk 1]
```diff
@@ -60,11 +60,15 @@ public function placeFromModel(){
 	 * Map center can be specified
 	 */
 	public function renderMarkers() {
+		// SUS-5128 | add Google Maps API key
+		global $egMapsGMaps3ApiKey;
```
Comment (mszabo-wikia) [inline extensions/wikia/Places/PlacesController.class.php]: Let's use wgGoogleMapsKey instead

**Review by mszabo-wikia — CHANGES_REQUESTED** [review]

File: extensions/wikia/Places/PlacesController.class.php (hunk 1 above)
Comment (macbre) [reply extensions/wikia/Places/PlacesController.class.php]: `egMapsGMaps3ApiKey` has different values for different environments while `wgGoogleMapsKey` is fixed.

Comment (mszabo-wikia) [reply extensions/wikia/Places/PlacesController.class.php]: let's address that then, egMapsGMaps3ApiKey is Maps extension specific var :)

Comment (macbre) [reply extensions/wikia/Places/PlacesController.class.php]: Tried wgGoogleMapsKey on devbox. It does not work - `egMapsGMaps3ApiKey` handles this environment.

Comment (mszabo-wikia) [reply extensions/wikia/Places/PlacesController.class.php]: but then we should add the proper env overrides for wgGoogleMapsKey and have egMapsGMaps3ApiKey just be a simple reassignment—if we ever change or sunset Maps extension, tight coupling to its internals will cause confusion

Comment (macbre) [reply extensions/wikia/Places/PlacesController.class.php]: See 4ae2620

**Review by mszabo-wikia — APPROVED** [review]

**Pull request merged**